### ISSUE TITLE
Final project catalogue structure and copy edits

### ISSUE THREAD

**Issue by vic-topcoder** [appirio-tech/connect-app#1200]
Hey @vikasrohit - we have the final version of the project catalogue. For the milestone release we want to start with only 1 QA product, so instead of the category card we want the QA card.

- [x] Change QA category card with Crowd Testing (https://connect.topcoder-dev.com/new-project/crowd_testing) - we have the content and icon to put as first-level
- [x] Update development category final cards to match

![screen shot 2](https://user-images.githubusercontent.com/8484061/30937837-dd4c8514-a38c-11e7-80f8-2f0958ec3f13.png)
Page title: Software Development projects
Subtitle: What kind of development do you need?

1. Front-end (leads to https://connect.topcoder-dev.com/new-project/frontend_dev)
Title: Front-end
Copy: Translate your designs into Web or Mobile front-end

2. Back-end & API (leads to https://connect.topcoder-dev.com/new-project/api_dev)
Title: Back-end & API
Copy: Build the server, DB, and API for your app

3. Development Iteration (leads to https://connect.topcoder-dev.com/new-project/generic_dev)
Title: Development Integration
Copy: Get help with any part of your app or software

**Comment by vikasrohit** [comment]
Done. However, I have few notes here:

1. It is not straightforward to interchange product category with product itself as we are now doing with QA category. It is not something hardcoded which we just copy from one place and paste at another. It has well defined hierarchy of product category and product. We have already added complexity by having few cards on project category to directly behave as product and others as project category. So, we should be careful about such changes, copy changes are okay but playing type of cards and their ids is risky.

2. For dev project category I have added the three products as new products and removed the old existing one `visual_prototype` because we are changing the URL (now it has `frontend_dev`) and copy both for front end product.
